src/ContosoTraders.Ui.Website/playwright.config.ts

```diff
@@ -11,11 +11,6 @@ export default defineConfig({
   /* Maximum time one test can run for. */
   timeout: 80 * 1000,
   expect: {
-    /**
```

Comment:
removed since we don't want to set defaults inside our config. See https://github.com/microsoft/create-playwright/pull/84.